ethereum/rpc/types/utils.go

```diff
@@ -168,7 +168,7 @@ func FormatBlock(
 		"miner":            validatorAddr,
 		"mixHash":          common.Hash{},
 		"difficulty":       (*hexutil.Big)(big.NewInt(0)),
-		"extraData":        "",
```

Comment:
didn't we change this recently? cc: @yihuang @thomas-nguy

Reply:
yeah I remember 
https://github.com/tharsis/ethermint/pull/204

Reply:
Was there a reason why it was changed to an empty string? In the ethereum docs its returned as `0x` on its empty form. The current version breaks https://github.com/graphprotocol/graph-node

Reply:
related to this discussion https://github.com/tharsis/ethermint/issues/69
Maybe you can ask @leejw51crypto about it